docs/content/manual/harvester-rancher/4678-load-balancer-different-roles-pool.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Create load balancer on different roles pools in RKE2 cluster
+---
+
+* Related issues: [#4678](https://github.com/harvester/harvester/issues/4678) [BUG] Unable to successfully create LoadBalancer service from Guest cluster post 1.2.1 upgrade
+
+
+
+## Category: 
+* Rancher 
+
+## Verification Steps
+1. Prepare three nodes Harvester and import with Rancher
+1. Provision a RKE2 guest cluster in Rancher
+1. Create the first pool, specify the `etcd` and `control plane` roles
+1. Create the second pool, specify the `work` role only
+1. Add the following cloud config in the `User Data` of each pool
+    ```
+    write_files:
+    - encoding: b64
+    content: {harvester's kube config, the cluster namespace should be same as the pool you created (base64 encoded)}
+    owner: root:root
+    path: /etc/kubernetes/cloud-config
+    permission: '0644'
+    ```
+1. Get the Harvester kubeconfig file, remember to add the namespace (should be the same with the guest cluster)
+    ```
+    contexts:
+    - name: "local"
+    context:
+    user: "local"
+    cluster: "local"
+    namespace: "default" ---------------------> Add this line
+    ```
+1. Output the Harvester kubeconfig into base64 format without new line
+    ```
+    cat local.yaml | base64 -w 0
+    ```
+1. Copy the base64 encoded kubeconfig to the cloud config write file sections above
+{{< image "images/rancher/4678-base64-kubeconfig.png" >}}
+1. Provision the RKE2 guest cluster
+1. After pools are created, we remove the harvester-cloud-provider in Apps > Installed Apps (kube-system namespace).
+1. Add new charts in Apps > Repositories, use https://charts.harvesterhci.io to install and select 0.2.3.
```

Comment:
To check a particular version of chart, removing the default and installing from Apps is fine but in general, this should not be practiced. As the bundled chart might get rolled back to the manifest shipped with guest cluster. So, I think adding this as generalized test case is not required.

Reply:
Thanks for the suggestion. 

That's true, the tester should confirm the cloud provider default provided in the RKE2 guest cluster at least to have `0.2.3` to include this changes. 
   
These steps here is not necessary since it would also exists at the time when the bundled chart is not ready. 
We can remove these lines and the related static screenshot. 
Then add the check step to ensure the cloud provider version.